marimo/_plugins/ui/_impl/table.py

```diff
@@ -181,6 +182,7 @@ def __init__(
         selection: Optional[Literal["single", "multi"]] = "multi",
         page_size: int = 10,
         show_column_summaries: bool = True,
+        format_mapping: Optional[Dict[str, str | Callable]] = None,
```

Comment:
this is great. i like the API, do we need to type `Callable`? `Callable[Any, str]`

Reply:
like `format_mapping: Optional[Dict[str, str | Callable[Any, Any]]] = None,`?

Reply:
yetp that looks good